torch/quantization/fx/quantization_patterns.py

```diff
@@ -1496,7 +1496,9 @@ def convert(self,
                 load_arg: Callable,
                 is_reference: bool = False,
                 convert_custom_config_dict: Dict[str, Any] = None) -> Node:
-        if is_reference:
+        # always produce reference pattern for relu
```

Comment:
Why should we do this? If we lower to fbgemm or qnnpack, isnt this incorrect?

Reply:
oh, this is to change convert_fx always to produce reference pattern, and we'll fuse the pattern in lower_to_fbgemm step

Reply:
would this break anything for customers who are not using the new `lower_to_fbgemm` function yet?

Reply:
no, this is internal implementation change, users are not explicitly calling lower_to_fbgemm currently. (they just need to call convert_fx)

Reply:
I'm probably confused, let's say a customer has a relu in their model, is_reference is False, and they are not using any lowering functions.  Would this PR change the behavior of calling convert on their model?  From the code it seems like yes, but I might be missing something.

Reply:
no, currently user do not need to call lowering function separately, it's called in convert_fx when is_reference=False.

Reply:
makes sense, thanks!